data/json/hobbies.json

```
@@ -91,5 +91,59 @@
     "description": "You enjoyed camping in the vast wilderness as a means of escaping the pace of modern life.  The wilderness isn't filled with people, so it must be safe.  Right?  It must be.",
     "points": 2,
     "skills": [ { "level": 2, "name": "survival" }, { "level": 2, "name": "firstaid" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "painting",
+    "name": "Painting",
+    "description": "You know your way around a canvas and a brush.  Fine motor coordination and basic applied chemistry come more naturally to you.",
+    "points": 2,
+    "skills": [ { "level": 2, "name": "chemistry" }, { "level": 2, "name": "cutting" } ]
```

```suggestion
    "skills": [ { "level": 1, "name": "fabrication" }, { "level": 1, "name": "traps" } ]
```
As an amateur painter I really don't think that weapon skills and applied science connect. However it does give you good fine motor skills.